Makefile

```diff
@@ -1,7 +1,9 @@
 export CARGO_PROFILE_RELEASE_CODEGEN_UNITS = 1
 export CARGO_PROFILE_RELEASE_LTO = fat
 export DOCKER_BUILDKIT = 1
-export RUSTFLAGS = -D warnings
+export CFLAGS = -mavx -mpopcnt -msse3 -mssse3 -msse4.1 -msse4.2 -msse4a -mcx16
+export CXXFLAGS = ${CFLAGS}
+export RUSTFLAGS = -D warnings --cfg near_production -Ctarget-feature=+avx,+cmpxchg16b,+popcnt,+sse3,+ssse3,+sse4.1,+sse4.2,+sse4a
```

Comment:
I wonder if we need the cfgs? When we control `near_production`, we also control cargo feature flags, so we *know* that `cpu_compatability_checks` is set. That is, perhaps just add `cpu_compatibility_checks` to `neard` crate?